.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,26 @@
+
+## #️⃣ 예상 결과
+
+> 예상했던 정상적인 결과가 어떤 것이었는지 설명해주세요.
+
+## #️⃣ 실제 결과
+
+> 실제로 어떤 결과가 발생했는지 설명해주세요.
+
+## 📎 참고할만한 자료(선택)
+
+> 관련 문서, 스크린샷, 또는 예시 등이 있다면 여기에 첨부해주세요
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)